scripts/e2e-tests.sh

```diff
@@ -261,6 +260,11 @@ function create_test_cluster_with_retries() {
   done
 }
 
+# Flag which project holds the actual e2e test cluster.
+# It will be a project assigned by Boskos if test is running on Prow, 
+# otherwise will be ${GCP_PROJECT} set up by user.
+E2E_CLUSTER_PROJECT=""
```

Comment:
Please put this declaration together with the other cars at the beginning of the file.

Reply:
Sorry I'm not quite familiar with shell. But it seems we'll need to `export` the var for it to be used by other bash subprocesses? I've made some changes, could you check if it is the right way to go?

Reply:
Yes, `export` would be good.